stormgls/stormgls.py

```diff
@@ -97,12 +108,66 @@ async def loadCompletions(self, core):
                 'help': '\n'.join(argp.mesgs)
             }
 
+    def cleanCheck(self, query):
```

Comment:
This is great! Maybe for a future change: it would be nice if some of this was configurable. Similar to the way pycodestyle (and most other linters) allow you to enable/disable certain lint checks. Probably not needed now but I can see this getting spammy on older storm packages.

Reply:
That sounds like a good idea. Does open up the whole question of what does configuring this look like, but that's a question we can cross when we get there (most likely just using mainline's config.py stuff or something since it's *right there*)